XCUITests/SaveLoginsTests.swift

```diff
@@ -44,79 +102,82 @@ class SaveLoginTest: BaseTestCase {
     }
 
-
     func testDeleteLogin() {
-        saveLogin()
+        saveLogin(givenUrl: testLoginPage)
         openLoginsSettings()
-        app.tables.cells[savedLoginEntry].tap()
+        app.staticTexts[domain].tap()
         app.cells.staticTexts["Delete"].tap()
         waitForExistence(app.alerts["Are you sure?"])
         app.alerts.buttons["Delete"].tap()
         waitForExistence(app.tables["Login List"])
-        XCTAssertFalse(app.tables.cells[savedLoginEntry].exists)
-        XCTAssertEqual(app.tables["Login List"].cells.count, 0)
-        XCTAssertTrue(app.tables["No logins found"].exists)
+        XCTAssertFalse(app.staticTexts[domain].exists)
+        XCTAssertFalse(app.staticTexts[domainLogin].exists)
+        XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList)
+        //Must've gone into a different view too
+       // XCTAssertTrue(app.staticTexts["No logins found"].exists)
     }
 
-
     func testEditOneLoginEntry() {
-        saveLogin()
+        saveLogin(givenUrl: testLoginPage)
         openLoginsSettings()
-        XCTAssertTrue(app.tables.cells[savedLoginEntry].exists)
-
-        app.tables.cells[savedLoginEntry].tap()
+        XCTAssertTrue(app.staticTexts[domain].exists)
+        XCTAssertTrue(app.staticTexts[domainLogin].exists)
+        app.staticTexts[domain].tap()
         waitForExistence(app.tables["Login Detail List"])
-        XCTAssertTrue(app.tables.cells["website, \(domain)"].exists)
-        XCTAssertTrue(app.tables.cells["username, test@example.com"].exists)
-        XCTAssertTrue(app.tables.cells["password"].exists)
+        XCTAssertTrue(app.tables.cells[loginsListURLLabel].exists)
+        XCTAssertTrue(app.tables.cells[loginsListUsernameLabel].exists)
+        XCTAssertTrue(app.tables.cells[loginsListPasswordLabel].exists)
         XCTAssertTrue(app.tables.cells.staticTexts["Delete"].exists)
     }
 
-
     func testSearchLogin() {
-        saveLogin()
+        saveLogin(givenUrl: testLoginPage)
         openLoginsSettings()
-
         // Enter on Search mode
-        app.otherElements["Enter Search Mode"].tap()
-        app.textFields["Search Input Field"].tap()
-
-        // Type Text that matches user, website, password
-        app.textFields["Search Input Field"].typeText("test")
-        XCTAssertEqual(app.tables["Login List"].cells.count, 1)
+        app.searchFields["Filter"].tap()
+        // Type Text that matches user, website
+        app.searchFields["Filter"].typeText("test")
+        XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsLoginsList + 1)
 
         // Type Text that does not match
         app.typeText("b")
-        XCTAssertEqual(app.tables["Login List"].cells.count, 0)
-        waitForExistence(app.tables["No logins found"])
+        XCTAssertEqual(app.tables["Login List"].cells.count, defaultNumRowsEmptyFilterList)
+        //Must've gone into a different view too
+        //waitForExistence(app.tables["No logins found"])
```

Comment:
This part is working on latest master, the `No logins found`message is shown. What we need to change is the assertion above because now the logins list contains two cells, with the options to Save Logins and Show In Application menu when the list is empty, so instead of 0 we need to equal to 2...